pkg/genlib/generator_interface.go

```diff
@@ -559,21 +531,22 @@ func bindLong(fieldCfg ConfigField, field Field, fieldMap map[string]any) error
 		return nil
 	}
 
+	min, _ := fieldCfg.Range.MinAsFloat64()
+	max, _ := fieldCfg.Range.MaxAsFloat64()
+
 	var emitFNotReturn emitFNotReturn
 	emitFNotReturn = func(state *GenState, buf *bytes.Buffer) error {
 		dummyInt := dummyFunc()
 		if previousDummyInt, ok := state.prevCache[field.Name].(int); ok {
-			adjustedRatio := rand.Float64() * (fuzzinessNumerator / fuzzinessDenominator)
-			if rand.Int()%2 == 0 {
-				adjustedRatio += 1.
-			} else {
-				adjustedRatio = 1. - adjustedRatio
-			}
-			dummyInt = int(math.Ceil(float64(previousDummyInt) * adjustedRatio))
+			lowerBound := float64(previousDummyInt) * (1 - fieldCfg.Fuzziness)
```

Comment:
I see this code repeated with small differences in multiple places. Can we extract it to a single function and performs only conversion where needed?